src/frontend/js/components/LTIConsumer/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { iframeResizer } from 'iframe-resizer';
+import { LTIConsumer as LTIConsumerProps } from 'types/LTIConsumer';
+
+export const LTIConsumer = ({ url, contentParameters }: LTIConsumerProps) => {
+  const formRef = React.useRef<HTMLFormElement>(null);
+  const iframeRef = React.useRef<HTMLIFrameElement>(null);
+
+  useEffect(() => {
+    formRef.current?.submit();
+    iframeResizer({ sizeWidth: true }, iframeRef.current!);
+  });
```

Comment:
```suggestion
  }, []);
```

You should specify an empty array as second argument of your `useEffect` hook elsewhere the function will be trigger at each renders. We would like to trigger it only when the component is mounted.